### PR TITLE
TNO-781: New toning group

### PR DESCRIPTION
**Pull request by jdtoombs**
image

Updated toning group on the content form.

### CODE REVIEW

File: app/editor/src/components/form/toning/ToningGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  fieldName: keyof IContentModel;
+}
+
+export const ToningGroup: React.FC<IToningGroupProps> = ({ fieldName }) => {
+  const toningOptions = [-5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5];
+  const { values, setFieldValue, errors, touched } = useFormikContext<IContentModel>();
+
+  const [active, setActive] = React.useState<number>();
+  React.useEffect(() => {
+    if (values.tonePools?.length && values.tonePools[0].value) setActive(values.tonePools[0].value);
```
Comment (Fosol) [inline app/editor/src/components/form/toning/ToningGroup.tsx:19]: defaults to `-5`?  Is this what we want?

File: app/editor/src/components/form/toning/ToningGroup.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        {toningOptions.map((option) => (
+          <Col>
+            {determineIndicator(option)}
+            <button
+              className={active === option ? 'active' : ''}
+              type="button"
+              key={option}
+              onClick={() => {
+                setActive(option);
+                setFieldValue('tonePool', option);
```
Comment (Fosol) [inline app/editor/src/components/form/toning/ToningGroup.tsx:52]: Why do we need to set two properties?

File: app/editor/src/components/form/toning/ToningGroup.tsx (hunk 1 above)
Comment (jdtoombs) [reply app/editor/src/components/form/toning/ToningGroup.tsx:18]: this is actually looking at the existing formik value of tonePools which the way we have it setup will always be tonePools[0]. it currently defaults to nothing when creating new content!

File: app/editor/src/components/form/toning/ToningGroup.tsx (hunk 2 above)
Comment (jdtoombs) [reply app/editor/src/components/form/toning/ToningGroup.tsx:51]: Can't remember the exact reasoning, but it was done this way in order to get the validation to work correctly

**Review by Fosol — APPROVED** [review]

**Pull request merged**